doc/mediasdk-man.md

```diff
@@ -1973,17 +1973,23 @@ This function initiates execution of an asynchronous function not already starte
 | | |
 --- | ---
 `MFX_ERR_NONE` | The function completed successfully.
+`MFX_ERR_NONE_PARTIAL_OUTPUT` | The function completed successfully, bitstream contains a portion of the encoded frame according to required granularity.
 `MFX_WRN_IN_EXECUTION` | The specified asynchronous function is in execution.
 `MFX_ERR_ABORTED` | The specified asynchronous function aborted due to data dependency on a previous asynchronous function that did not complete.
 
 **Change History**
 
 This function is available since SDK API 1.0.
+Return code MFX_ERR_NONE_PARTIAL_OUTPUT is introduced in SDK API TBD.
 
 **Remarks**
 
 See status codes for specific asynchronous functions.
 
+In partial output mode, enabled during encoder initialization [mfxExtPartialBitstreamParam](#mfxExtPartialBitstreamParam), application is expected to call SyncOperation repeatedly until it receives the MFX_ERR_NONE (or some error);
```

Comment:
repeatedly with which sync point, the same? I think it worths to be said explicitly in the man.

If library changes mfxBitstream in async part, then there will be race condition between library async part and end-user application. How user is supposed to handle it having no mutex to lock?

Reply:
- We reuse the same sync point.
- Library update the app provided bitstream structure only during the SyncOperation call, this should be explicitly documented (any processing of the partial output on app level must be finished before next SyncOperation call). Good catch.

Reply:
I guess the same possibility for race condition is true for any other bitstream changes in library and end-user application. Nothing special here.

Reply:
No. If SyncOperation returned MFX_ERR_NONE then library is done with the bitsteam and won't modify it. I guess that what you really want to say is that partial bitstream handling will work in a very special mode only (external threading?), i.e. when there is synchronous processing and library can't actually change the bitstream while user works with it. I think that's what needs to be clarified.

Reply:
But if it returns MFX_WRN_IN_EXECUTION then SynOperation is incomplete and there is possibility to change bitstream simultaneously in library and from end-user side from different threads.

Reply:
I guess also if end-users uses one mfxBistream for multiple frames sinking data after each frame it is possible to have race conditions because bitstream modification happens on event from hw (or scheduler in single thread mode) that can happen during end-user bitstream modification.